guides/v2.1/cloud/project/project-routes-more-ssi.md

```diff
@@ -8,7 +8,9 @@ functional_areas:
 
 [Server side includes](http://httpd.apache.org/docs/current/howto/ssi.html){:target="_site"} (SSI) are directives in [HTML](https://glossary.magento.com/html) pages that get evaluated on the server while the pages are being rendered. Use of [server side](https://glossary.magento.com/server-side) includes enables you to add dynamically generated content to an existing HTML page without having to serve the entire page.
 
-More information about [nginx SSI](http://nginx.org/en/docs/http/ngx_http_ssi_module.html).
+For more information about [nginx SSI](http://nginx.org/en/docs/http/ngx_http_ssi_module.html).
```

Comment:
```suggestion
See [nginx SSI](http://nginx.org/en/docs/http/ngx_http_ssi_module.html).
```

Reply:
Earlier reference goes to Apache docs, I replaced that one with the NGINX reference, and removed the "See nginx SSI." reference."

Reply:
Good idea.